ui/mods/msu/utilities.js

```diff
@@ -0,0 +1,97 @@
+var MSU = {};
+MSU.printData = function( _obj, _depth, _maxLen )
+{
+	if (_depth === undefined) _depth = 1;
+	if (_maxLen === undefined) _maxLen = 0;
+	var length = 1;
+	var isArray = false;
+	if (typeof _obj == "object" && _obj !== null)
+	{
+		if (Array.isArray(_obj))
+		{
+			isArray = true;
+			length = _obj.length;
+		}
+		else
+		{
+			length = Object.keys(_obj).length;
+		}
+	}
+	var string = MSU.getLocalString("Data: ", _obj, Math.max(length, _maxLen), _depth, isArray);
+	for (var i = 0; i < string.length; i = i + 900)
```

Comment:
This is necessary because for some reason console.error has a really shorter character limit

Reply:
Mark as Resolved?